kube-runtime/src/wait.rs

```diff
@@ -214,6 +215,23 @@ pub mod conditions {
         }
     }
 
+    /// An await condition for `Job` that returns `true` once it is completed
+    #[must_use]
+    pub fn is_job_completed() -> impl Condition<Job> {
```

Comment:
Should this be in a separate PR? It won't be listed in the changelog.

Reply:
I can factor it into its own PR.

Reply:
https://github.com/kube-rs/kube-rs/pull/935. Let's do that one first.